cost-analyzer/templates/cost-analyzer-deployment-template.yaml

```diff
@@ -307,7 +307,11 @@ spec:
             - name: CLOUD_PROVIDER_API_KEY
               value: "AIzaSyDXQPG_MHUEy9neR7stolq6l0ujXmjJlvk" # The GCP Pricing API requires a key.
             - name: GOOGLE_APPLICATION_CREDENTIALS
+            {{- if .Values.kubecostProductConfigs.gcpSecretName }}
+              value: /models/key.json
```

Comment:
Does this mean that `.Values.kubecostProductConfigs.gcpSecretName` existing means that a key gets mounted to `/models/key.json`?